File: Practica/Ej6.py
#Determinar si un numero es primo
n=int(input("Introduce el numero: "))
c=0; i=1
while i <=n:
    if n%i==0:
        c=c+1
    i=i+1
if c==2:
    print (n,"es primo")
else:
    print (n,"no es primo")

File: Practica-CodigosJava/Ej6.java
//Determinar si un numero es primo
import java.util.Scanner;
public class Ej6 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Introduce el número: ");
        int n = scanner.nextInt();
        int c = 0;
        int i = 1;
        while (i <= n) {
            if (n % i == 0) {
                c++;
            }
            i++;
        }
        if (c == 2) {
            System.out.println(n + " es primo");
        } else {
            System.out.println(n + " no es primo");
        }
        scanner.close();
    }
}
